ocrd_models/ocrd_models/ocrd_mets.py

```diff
@@ -210,8 +210,8 @@ def file_groups(self):
         """
 
         # WARNING: Actually we cannot return strings in place of elements!
-        #if self._cache_flag:
-        #    return self._file_cache.keys()
+        if self._cache_flag:
+           return list(self._file_cache.keys())
```

Comment:
This returns a list of the keys() of the `_file_cache` and the keys are of type `String` not of type `el_fileGrp`.